wiki/Tournaments/CMC/2020/en.md

```diff
@@ -0,0 +1,388 @@
+| Lower | Team 3 | 2 | **6** | Leviathan | [#1](https://osu.ppy.sh/community/matches/70941169) |
+
+Saturday, 5 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | L Hunter | 2 | **6** | **Team 11** | [#1](https://osu.ppy.sh/community/matches/70998868) |
+| Upper | SCP Indonesia | 2 | **6** | **UwU Team** | [#1](https://osu.ppy.sh/community/matches/71003960) |
+| Upper | **Neppu-Neppu** | **6** | 3 | GWS | [#1](https://osu.ppy.sh/community/matches/71008403) |
+
+Sunday, 6 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | GWS | 2 | **6** | Team 11 | [#1](https://osu.ppy.sh/community/matches/71071493) |
+| Lower | SCP Indonesia | 0 | **6** | **Leviathan** | *win by default* |
+
+### Quarterfinals
+
+Friday, 27 November 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | **Sumber Rejeki** | **5** | 0 | Do'a Ibu | [#1](https://osu.ppy.sh/community/matches/70548959) |
+
+Saturday, 28 November 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | Mak Lo Gamink | 0 | **5** | **Team 11** | [#1](https://osu.ppy.sh/community/matches/70605058) |
+| Lower | Team 16 | 0 | **5** | **Soundtoloyo** | [#1](https://osu.ppy.sh/community/matches/70600889) |
+| Upper | Leviathan | 2 | **5** | **Neppu-Neppu** | [#1](https://osu.ppy.sh/community/matches/70605490) |
+| Upper | Team 1 | 0 | **5** | **SCP Indonesia** | [#1](https://osu.ppy.sh/community/matches/70598560) |
+| Lower | **Pecel Lele Ujang** | **5** | 1 | Anti LN Club | [#1](https://osu.ppy.sh/community/matches/70601346) |
+| Upper | L Hunter | 4 | **5** | **UwU Team** | [#1](https://osu.ppy.sh/community/matches/70611342) |
+| Upper | Team 3 | 2 | **5** | **GWS** | [#1](https://osu.ppy.sh/community/matches/70601710) |
+
+Sunday, 29 November 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | Team 1 | 0 | **5** | **Team 11** | [#1](https://osu.ppy.sh/community/matches/70660894) |
+| Lower | **L Hunter** | **5** | 0 | Pecel Lele Ujang | [#1](https://osu.ppy.sh/community/matches/70666560) |
+| Lower | **Team 3** | **5** | 2 | Soundtoloyo | [#1](https://osu.ppy.sh/community/matches/70600889) |
+| Lower | **Leviathan** | **5** | 0 | Sumber Rejeki | [#1](https://osu.ppy.sh/community/matches/70663842) |
+
+## Ruleset
+
+### General rules
+
+1. Map scoring is based on **Score V2.**
+2. The mapsets for each round will be announced by the Tournament Management in advance before the actual matches take place.
+3. Match schedules will be predetermined by the Tournament Management. Had there been any team(s) who were unable to attend the current schedule for any reason, all other affected teams may apply and settle for a reschedule at the #reschedule channel in the tournament's Discord server.
```

Comment:
this should prob keep consistent tense; it flip flops between present and past. this goes for the rest of the list as well